src/lib/docs/Utilities/SigningRequest.md

```diff
@@ -9,20 +9,28 @@ order: 5
 
 # Signing Request
 
-Signing Requests are a data type used in the Antelope ecosystem to facilitate the seamless transfer of transaction details in a compressed string format between applications and signature providers. Designed to encapsulate and transport transaction data, they can be shared through various methods, such as QR codes and hyperlinks.
+Signing Requests are a data type used in the Antelope ecosystem to facilitate the transferring transaction related data between applications for the purposes of generating a signature. This is done using a compressed string format designed to encapsulate and make it easier to transport data, which can be communicated through various methods such as QR codes and hyperlinks.
```

Comment:
`the transferring of transaction related data`?

Reply:
"the transferring OF" - the "of" was missing 😅